src/plugins/controls/public/control_group/embeddable/control_group_container.tsx

```diff
@@ -11,6 +11,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { compareFilters, COMPARE_ALL_OPTIONS, Filter, uniqFilters } from '@kbn/es-query';
 import { BehaviorSubject, merge, Subject, Subscription } from 'rxjs';
+import _ from 'lodash';
```

Comment:
Honest question, because I really don't know. Would `import { isEqual } from 'lodash';` be any better for compilation or performance?

Reply:
Makes sense. Looks like we also have lodash in [@kbn/ui-shared-deps-src](https://github.com/elastic/kibana/blob/main/packages/kbn-ui-shared-deps-src/src/entry.js#L48). And I've found [more context here](https://github.com/elastic/kibana/pull/78100). tldr; don't import `from 'lodash/*`.